packages/perseus-core/src/feature-flags.ts

```diff
@@ -8,6 +8,6 @@
  * 3. When no active feature flag is available value will be:
  *    export const PerseusFeatureFlags = [];
```

Comment:
Can you add the `as const` to the example just so somebody doesn't accidentally remove it when they're following the instructions.